File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1052_basico.js
import { question } from "readline-sync";

function main() {
    const MOUNTH = get_number();
    let menssage;

    menssage = (MOUNTH === 1) ? `January` : (MOUNTH === 2) ? `February` : (MOUNTH === 3) ? `March` : (MOUNTH === 4) ?
        `April` : (MOUNTH === 5) ? `May` : (MOUNTH === 6) ? `June` : (MOUNTH === 7) ? `July` : (MOUNTH === 8) ? `August` :
            (MOUNTH === 9) ? `September` : (MOUNTH === 10) ? `October` : (MOUNTH === 11) ? `November` : `December`;

    print(menssage);
}

function get_number() {
    return parseInt(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1052_basico.cpp
#include <stdio.h>
#include <iostream>
#include <string.h>
using namespace std;

main() {
    int mes_Numerico;
    cin >> mes_Numerico;
    string mes;

    if (mes_Numerico == 1) {
        mes = "January";
    } else if (mes_Numerico == 2) {
        mes = "February";
    } else if (mes_Numerico == 3) {
        mes = "March";
    } else if (mes_Numerico == 4) {
        mes = "April";
    } else if (mes_Numerico == 5) {
        mes = "May";
    } else if (mes_Numerico == 6) {
        mes = "June";
    } else if (mes_Numerico == 7) {
        mes = "July";
    } else if (mes_Numerico == 8) {
        mes = "August";
    } else if (mes_Numerico == 9) {
        mes = "September";
    } else if (mes_Numerico == 10) {
        mes = "October";
    } else if (mes_Numerico == 11) {
        mes = "November";
    } else if (mes_Numerico == 12) {
        mes = "December";
    }
    cout << mes.c_str() << endl;
}